fdbclient/S3Client.actor.cpp

```diff
@@ -127,10 +127,7 @@ Reference<S3BlobStoreEndpoint> getEndpoint(const std::string& s3url,
 			throw backup_invalid_url();
 		}
 
-		if (resource.empty()) {
-			TraceEvent(SevError, "S3ClientGetEndpointEmptyResource").detail("URL", s3url);
```

Comment:
What if the resource is empty when doing a real bulkload job? Thanks!

Reply:
Same thing as happens when you provide any bad URL -> the job submits and then the job keeps failing because no such entity up in s3.